README.md

```diff
@@ -1,9 +1,3 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
 ## Getting Started
 
 This repository includes files with plain SQL that can be used to recreate a database:
```

Comment:
- [ ] Please remove this section, because it is unnecessary or redundant. Your `README` should be professional and free from any kind of error, as you can also notice your `getting-started` section is already added below the `key features`. So that's why don't need it here, this particular place is actually for the `logo` for your project.